client/task-list/style.scss

```diff
@@ -40,8 +40,8 @@
 		}
 	}
 
-	.woocommerce-task__estimated-time {
-		color: $core-grey-dark-300;
+	.woocommerce-task-estimated-time {
```

Comment:
This has broken things a little - replacing the `__` with a single dash:
![image](https://user-images.githubusercontent.com/224531/87505440-368b6100-c6ac-11ea-9f00-ac40c20c4bb7.png)

Reply:
oof thank you for spotting this! will fix.